### ISSUE TITLE
Add dark theme colors for modal

### ISSUE THREAD

**Issue by draperunner** [entur/shamash#27]
If you click "Search for ID" a modal opens. This modal has only light theme support. It should also have a dark theme.

The exact colors are not important. We don't follow a strict color palette (yet). Just pick some dark colors for the background and light colors for the foreground. :D

![image](https://user-images.githubusercontent.com/4339443/66243135-b6d76b80-e703-11e9-9eb3-8e5656f65361.png)

**Comment by dgupta04** [comment]
Hey, I'd like to give this a shot!

**Comment by draperunner** [comment]
Great!

**Comment by dgupta04** [comment]
I had a doubt, should I create separate CSS classes for the dark theme or modify already written classes?

**Comment by draperunner** [comment]
Since the addition of `utils/getPreferredTheme` (#29) I think you can do something like this:

Get the theme using `getPreferredTheme`:
```
function GeocoderModal({ onDismiss }) {
    const theme = getPreferredTheme()
```

Switch classes based on theme:
```
<div className={`geocoder-modal--${theme}`} 
```

And then update styles:
```
.geocoder-modal {
    border-radius: 4px;
    padding: 1rem;
    position: absolute;
    width: 50vw;
    min-width: 500px;
    left: 25vw;
    top: 100px;
    z-index: 10;
}
.geocoder-modal--light {
    background-color: white;
}
.geocoder-modal--dark {
    background-color: black;
}
```

**Comment by dgupta04** [comment]
Oh alright, I'll implement it that way. Thanks!

**Comment by draperunner** [comment]
Fixed by #30
